packages/block-library/src/video/edit.js

```diff
@@ -129,6 +129,7 @@ class VideoEdit extends Component {
 			poster,
 			preload,
 			src,
+			playsinline,
```

Comment:
Can you update this variable/attribute name to be written as `playsInline`? I think it is what React expects and it also reads easier that way.

Reply:
Thank you so much @gziolo ❤️ Updated [here](https://github.com/WordPress/gutenberg/pull/14500/commits/23cf5beb2b074fb6daa9de96f0de38f86c5a857a).